src/core/File.ts

```diff
@@ -159,10 +161,14 @@ export class File {
         if (info.params) {
             this.params = info.params;
         }
+
         this.absPath = info.absPath;
+
         if (this.absPath) {
             this.relativePath = ensureFuseBoxPath(path.relative(this.context.appRoot, this.absPath));
         }
+
+        this.context.extensionOverrides.setOverrideFileInfo(this);
```

Comment:
What is it doing in alternative content? Shouldn't it be at `consume` level? That one limits many plugins

Reply:
It doesn't just do this in alternative content. We do this work in the `File` constructor so it happens at the lowest level possible. All testing I have done doesn't look to have impacted.

Reply:
Then why is it happening on `addAlternativeContent` ?

Reply:
Its not?
<img width="718" alt="screen shot 2017-10-16 at 20 43 47" src="https://user-images.githubusercontent.com/3731526/31631804-bfd0af62-b2b2-11e7-982d-bcfd83e11b61.png">

Reply:
sorry man, i am tired ;-))

Reply:
hehe no worries bud.